__tests__/app.test.js

```diff
@@ -11,4 +13,61 @@ describe('backend-anyapi routes', () => {
   afterAll(() => {
     pool.end();
   });
+
+  it('should create an order', async () => {
+    const res = await request(app)
+      .post('/api/v1/onepiece')
+      .send({ name: 'Monkey D. Luffy', crew: 'Straw Hats' });
+
+    expect(res.body).toEqual({
+      id: expect.any(String),
+      name: 'Monkey D. Luffy',
+      crew: 'Straw Hats',
+    });
+  });
+
+  it('Should be able to list characters', async () => {
+    await insert({ name: 'Monkey D. Luffy', crew: 'Straw Hats' });
```

Comment:
We might consider inserting multiple characters to ensure that our getAll route returns all of them